src/generate.js

```diff
@@ -94,7 +94,7 @@ export const defaultSelectorHandlers = [
         }
         // Generate the styles normally, and then wrap them in the media query.
         const generated = generateSubtreeStyles(baseSelector);
-        return `${selector}{${generated}}`;
+        return [`${selector}{${generated}}`];
```

Comment:
`generateSubtreeStyles` should have the same return type as `generateCSS` so `pseudoSelectors` should already return an array. Would it make sense to make the Flow type for `generateSubtreeStyles` more specific?

We could have `generateCSS` handle a string being returned here, but we never want a selector handler to return strings containing multiple rules, as `insertRule` expects a single rule to be passed. My thought here was that selector handlers should always return an array of strings with one rule per string.

This would be a breaking change to the current API, so perhaps we could, say, wrap strings returned by a selector handler in a `@media all {...}` query and warn about that return type being deprecated. Or just bump by a major version.